glottolog3/__init__.py

```diff
@@ -117,6 +117,11 @@ def main(global_config, **settings):
         '/glottolog',
         views.languages,
         renderer='language/search_html.mako')
+    config.add_route_and_view(
+        'glottolog.bpsearch',
+        '/bp/search',
+        views.languages,
```

Comment:
In a future PR we'll want to create a new view for this.

Reply:
noted